.github/workflows/psalm.yml

```diff
@@ -13,17 +13,25 @@ jobs:
         uses: shivammathur/setup-php@v2
         with:
           php-version: '8.0'
-          extensions: "json,couchbase,memcached,mongodb,redis,xsl,ldap,dom"
+          extensions: "json,memcached,mongodb,redis,xsl,ldap,dom"
           ini-values: "memory_limit=-1"
           coverage: none
 
-      - name: Checkout code
+      - name: Checkout base
         uses: actions/checkout@v2
+        with:
+          path: pr
+
+      - name: Checkout PR
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.base_ref }}
```

Comment:
would something like this be more accurate?
`git merge-base --fork-point ${{ github.base_ref }} ${{ github.ref }}`

Reply:
We are currently not on a git repository (Nothing is checked out)

Also note: 
```
${{ github.base_ref }} // 5.x
${{ github.ref }} // refs/pull/40291/merge
```